### ISSUE TITLE
Add text reflow after zooming to ChangeLog

### ISSUE THREAD

**Issue by yvolk** [andstatus/andstatus#476]
Since some KitKat Android update text in WebViews doesn't reflow after soom.
See https://stackoverflow.com/questions/19986305/no-more-text-reflow-after-zoom-in-kitkat-webview

**Comment by yvolk** [comment]
After reading different texts on this, and trying solutions, including https://stackoverflow.com/questions/19986305/no-more-text-reflow-after-zoom-in-kitkat-webview I see that text reflow doesn't work good.
But I found a solution that is consistent with existing Appearance->Size settings in AndStatus: text zoom in the WebViews is set to different values, dependending on selected Size. Thus text size in the ChangeLog changes in a close sync with text size changes in a Timeline.
